app/src/main/java/org/mozilla/fenix/compose/Divider.kt

```diff
@@ -0,0 +1,154 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Generic Divider
+ *
+ * @param modifier [Modifier] Used to shape and position the underlying [androidx.compose.material.Divider]
+ * @param color  [Color] of the divider line
+ */
+@Composable
+fun Divider(
+    modifier: Modifier = Modifier,
+    color: Color = FirefoxTheme.colors.borderPrimary,
+) {
+    androidx.compose.material.Divider(
+        modifier = modifier,
+        color = color,
+        thickness = 0.5.dp,
```

Comment:
I think this was my fault. In https://mozilla.slack.com/archives/G01EXNFFAVC/p1666196929822729?thread_ts=1666195522.000849&cid=G01EXNFFAVC, I mentioned that we can replace the Divider in https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt#371 with a thickness of 1dp. See the figma in the slack message https://www.figma.com/file/am9EJGg6ow29PXWkc9jcOH/MR2-(2021)?node-id=3022%3A48384.

Since 1.0dp thickness is the default for the material design divider, we should also be able to just remove this line.

Reply:
ah ok I'll bring back the default thickness for the Divider

Reply:
there are currently 2 dividers with thicknesses of "0.5.dp"
- `TabGridItem.kt` line 162
- `RecentlyVisited.kt` line 367

should those be changed to 1.dp or should I refactor the `Divider` to pass the thickness forward to the Material Compose Divider?

@gabrielluong

Reply:
Oh in that case, let's leave `thickness` as an option for `TabGridItem`. We could look into some figma to see if that thickness should also be updated, but I think that could be done as followup.

I think for `RecentlyVisited` we can already convert this to 1dp since we can see 1dp thickness in the designs for https://www.figma.com/file/am9EJGg6ow29PXWkc9jcOH/MR2-(2021)?node-id=3022%3A48384. However, I would suggest testing this and confirming what you see matches the figma designs.

Reply:
actually nevermind, the one in TabGridItem is 1.dp so I can go ahead and refactor that